src/app/report/edit-display/edit-display.component.ts

```diff
@@ -245,14 +202,11 @@ export class EditDisplayComponent {
     });
   }
 
-  closeModal(): void {
-    this.modalService.dismissAll();
-  }
-
   copyReport(): void {
-    const storageId: number = this.report.xml ? +this.report.storageId : +this.report.uid.split('#')[0];
-    const data: any = {};
-    data[this.currentView.storageName] = [storageId];
+    const storageId: number = this.report.storageId ?? +this.report.uid.split('#')[0];
+    const data: any = {
```

Comment:
This could be set to a record<string,number> right?

Reply:
fixed, also fixed it in another component